code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Toxins.dm

```diff
@@ -156,6 +156,58 @@
 	T.assume_gas(GAS_PHORON, amount, T20C)
 	remove_self(amount, holder)
 
+/decl/reagent/toxin/phoron/kois //and it infects the phoron at a molecular level and its half demon and angel and half werewolf so you cant get rid of it
+	description = "Phoron in its liquid form. Twice as potent when breathed in. Contains biological traces."
+	var/kois_type = 1
+
+/decl/reagent/toxin/phoron/kois/black
+	description = "Phoron in its liquid form. Twice as potent when breathed in. Contains unknown biological traces."
+	var/kois_type = 2
+
+/decl/reagent/toxin/phoron/kois
+	/decl/reagent/toxin/phoron/kois/affect_ingest(var/mob/living/carbon/human/M, var/alien, var/removed, var/datum/reagents/holder)
+		if(!ishuman(M))
+			return
+		var/is_vaurcalike = (alien == IS_VAURCA)
+		if(!is_vaurcalike)
+			var/obj/item/organ/internal/parasite/P = M.internal_organs_by_name["blackkois"]
+		if(istype(P) && P.stage >= 3)
+			is_vaurcalike = TRUE
+		if(is_vaurcalike)
+			M.heal_organ_damage(1.4 * removed, 1.6 * removed)
+			M.adjustNutritionLoss(-nutriment_factor * removed)
+		else
+			infect(M, alien, removed)
+
+	/decl/reagent/toxin/phoron/kois/affect_blood(var/mob/living/carbon/M, var/alien, var/removed, var/datum/reagents/holder)
```

Comment:
This and all the procs below are indented one to the right too much.

Reply:
I did do that, I think I fixed it